Control/Monad/Except.hs

```diff
@@ -38,6 +38,17 @@ module Control.Monad.Except
     -- * Monads with error handling
     Error.MonadError(..),
     Error.liftEither,
+    -- * The ExceptT monad transformer
```

Comment:
How does this render in haddock?  It seems that `tryError` and the like would belong to `MonadError` rather than `Except`.
Confer this with what it was previously: https://github.com/haskell/mtl/blame/ecb525a5ba3521d0059eb2396a376b2e91557cff/Control/Monad/Except.hs#L39

I think the easiest path would be to restore the order to what it was before `ExceptT` was removed.

Reply:
This has now been done.